File: parent-group4/pingyougou-interface/src/main/java/com/itheima/core/ContentService.java
package com.itheima.core;

import com.itheima.core.pojo.ad.Content;
import com.itheima.core.pojo.good.Brand;
import com.itheima.core.pojo.item.ItemCat;
import entity.PageResult;

import java.util.List;

public interface ContentService {

	public List<Content> findAll();
	
	public PageResult findPage(Content content, Integer pageNum, Integer pageSize);
	
	public void add(Content content);
	
	public void edit(Content content);
	
	public Content findOne(Long id);
	
	public void delAll(Long[] ids);

    List<Content> findByCategoryId(Long categoryId);

	List<ItemCat> findByParentId();

	List<Brand> findBrandName();
}

File: parent-group4/pingyougou-web-portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){
	this.findByCategoryId = function(categoryId){
		return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
	}

    this.findByParentId = function(){
        return $http.get("content/findByParentId.do");
    }

    this.findBrandName=function () {
        return $http.get('../content/findBrandName.do');
    }
});